README.md

```diff
@@ -19,6 +19,9 @@ The three main goals are:
 * miriad (aipy) -> uvfits
 * FHD -> uvfits
 * FHD -> miriad (aipy)
+* CASA measurement sets -> miriad
+* CASA measurement sets -> FHD
```

Comment:
no loop-back tests exist for .ms->FHD. I've added "(untested)"
 in the README.md